src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/net45/TokenCachePlugin.cs

```diff
@@ -33,14 +33,14 @@
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
-    internal class TokenCachePlugin : ITokenCachePlugin
+    internal class TokenCachePlugin
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS1118)